client/app/routes/dashboards/dashboard.js

```diff
@@ -590,6 +592,27 @@ export default Ember.Route.extend({
                         ],
                     },
                     {
+                        widgetType: "stacked-bars",
+                        name: "Types",
```

Comment:
Let's move the types widget above the "Top Contributors" widget to match the dashboard mock-up